packages/data/index.js

```diff
@@ -10,6 +10,9 @@ module.exports = {
   options: {
     autoImport: {
       exclude: ['papaparse'] // only included during development
+    },
+    babel: {
+      plugins: ['@babel/plugin-proposal-optional-chaining']
```

Comment:
Add to `dependencies`/install

Reply:
Ignore for now, we should be able to remove with a newer version of babel

Reply:
https://github.com/yahoo/navi/pull/702